api/types/clusterconfig_override.go

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
Can't comment on the file name, but we don't usually use underscores in filenames (unless it's _test.go) so let's just name this clusterconfigoverride.go